src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Machine started its work.");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Machines stopped working.");

```

Comment:
Similarly, the message should specify that it is a `Truck` that has stopped working. Update the message to reflect this.